pkg/kubernetes/views/imagestream-body.html

```diff
@@ -1,3 +1,16 @@
+<div ng-repeat="statustags in stream.status.tags">
+  <div ng-repeat="condition in statustags.conditions" ng-if='condition.status == "False"'
+      class="alert alert-danger alert-dismissable">
+    <button type="button" class="close" data-dismiss="alert" aria-hidden="true">
+      <span class="pficon pficon-close"></span>
+    </button>
+    <span class="pficon pficon-error-circle-o"></span>
+    <strong>{{ condition.reason }} on {{ condition.type }}:</strong> {{ condition.message }} {{ condition.lastTransitionTime }}
+    <p ng-if='$first'>
+      <a translateable="yes" ng-click="modifyImageStream(stream)" class="alert-link">Edit imagestream</a>
```

Comment:
Well, there aren't a lot of options for any error. Our choices are pretty much limited to delete and edit.